main.js

```diff
@@ -0,0 +1,157 @@
+// 1 //
+
+const validity = function (value) {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return String(value);
+};
+
+const autoTeam = function (
+  teamName,
+  driverName,
+  yearBirth,
+  brandCar,
+  sponsor,
+  driverAcess
+) {
+  const newAutoTeam = {
+    teamName: validity(teamName),
+    driverName: validity(driverName),
+    yearBirth: validity(yearBirth),
+    brandCar: validity(brandCar),
+    sponsor: validity(sponsor),
+    driverAcess: driverAcess,
+  };
+  return newAutoTeam;
+};
+
+autoTeam("Valhalla", "Ulrih", 1980, "Volvo", "Norway Ragnarok", "yes");
+
+// 2 //
+
+const empty = "empty__field";
+
+const newUser = function () {
+  const user = {
+    login: ``,
+    password: ``,
+    city: ``,
+    country: ``,
+    maleOrFemale: ``,
+    age: ``,
+  };
+
+  for (const [key] of Object.entries(user)) {
+    const answer = prompt([key]);
+    if (answer === null || answer === ``) user[key] = empty;
+    else {
+      user[key] = answer;
+    }
+  }
+  return user;
+};
+const userCreated = newUser();
+console.log(userCreated);
+
+// 3 //
+
+const changeDATA = function (Object, key, value) {
+  return (Object[key] = value);
+};
+changeDATA(userCreated, "city", "Kherson");
+console.log(userCreated);
+
+// 4 //
+
+const hiEveryone = function () {
+  return console.log("Hi. everyone!");
+};
+
+const sendMyHomeWork = function () {
+  return console.log("Sending my howework... Please, wait");
+};
+
+const newStudent = {
+  name: "Victor",
+  surname: "Romaniv",
+  age: 29,
+  course: 3,
+  city: "Kyiv",
+  greeting: hiEveryone(),
+  addHomework: sendMyHomeWork(),
+};
+
+newStudent.greeting;
+newStudent.addHomework;
+
+// 5 //
+
+function isEmpty(obj) {
+  const counter = 0;
```

Comment:
```suggestion
  let counter = 0;
```
You updating value of the variable below